dm/tests/shardddl1/run.sh

```diff
@@ -406,7 +413,7 @@ function DM_UpdateBARule_CASE() {
 
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"show-ddl-locks" \
-		"\"ID\": \"test-\`shardddl\`.\`tb\`\"" 1
+		"no DDL lock exists" 1
```

Comment:
why this is no lock 😵‍💫

Reply:
No DDL infos in DM master because we remove all of them when stop-task. So no lock exists when task starts until master receive first DDL info.